moveit_ros/planning/robot_model_loader/src/robot_model_loader.cpp

```diff
@@ -105,7 +122,8 @@ void RobotModelLoader::configure(const Options& opt)
     moveit::tools::Profiler::ScopedBlock prof_block2("RobotModelLoader::configure joint limits");
 
     // if there are additional joint limits specified in some .yaml file, read those in
-    ros::NodeHandle nh("~");
+    // TODO (anasarrak): Add the correct node name to the .yaml
+    auto additional_joints_parameters = std::make_shared<rclcpp::SyncParametersClient>(node_);
```

Comment:
I don't see a reason to always initialize the `SyncParametersClient` for simple parameter lookup. We only use it for `has_parameter()` which is also provided by the node. Most queries here (and in the other classes) could be implemented using `bool Node::get_parameter()`, no? Especially in cases like here, we could always use the templated version of `get_parameter()` or `get_parameter_or()`. We only need to adjust for variable parameter types or catch the `rclcpp::ParameterTypeException`.